lstchain/data/lstchain_standard_config.json

```diff
@@ -145,18 +141,22 @@
        "sample_size": 10000,
        "sample_duration":100000,
        "tel_id":1,
+       "time_sampling_correction_path": null,
        "charge_product":"LocalPeakWindowSum",
        "charge_median_cut_outliers": [-0.5,0.5],
        "charge_std_cut_outliers": [-10,10],
        "time_cut_outliers": [2,38]
      },
     "LocalPeakWindowSum":{
       "window_shift": 5,
-      "window_width":12
+      "window_width":12,
+      "apply_integration_correction": false
      },
     "FixedWindowSum":{
-       "window_start": 12,
-       "window_width":12
+       "window_shift": 6,
+       "window_width":12,
+       "peak_index": 18,
```

Comment:
where is this `peak_index` coming from?

Reply:
ctapipe changed the options to the extractors, instead of window start / end  you now have peak_index / shift